textRevers/src/main/java/app/textRevers/CharsCounter.java

```diff
@@ -0,0 +1,79 @@
+package app.textRevers;
+
+import java.util.*;
+
+/**
+ * @author Anastasiia Nudha
+ * @version 0.1.0
+ * @since 0.0.1
+ */
+
+public class CharsCounter {
+
+    private static Map<String, String> cache = new HashMap<>();
+
+    /**
+     * Method name: countChars
+     *
+     * @param sentence The sentence in which be counted chars.
+     * @return (String) String display result.
+     * 											Inside the function: 1. Checking for null argument exception;
+     * 											2. Cache the result to cache map if the argument wasn't counted previously;
+     * 											3. Call the method countValue for adding to cache; 4. Return counted chars
+     * 											in String from sentence ;
+     */
+
+    public static String countChars(String sentence){
```

Comment:
Name does not response it`s body

Reply:
Why CountChars return String??? I thought count means calculate numbers.

Please stop looking for quick solutions to problems. Sit down, think about what I’m writing to you, let’s not do everything quickly, in case Victor screws up. This is not how a programmer should solve problems...